JSDemos/Demos/DataGrid/ToolbarCustomization/description.md

```diff
@@ -1,12 +1,10 @@
-The DataGrid uses an integrated DevExtreme [Toolbar](/Demos/WidgetsGallery/Demo/Toolbar/Overview/) component to display predefined and custom commands. To add or remove toolbar items, implement the [onToolbarPreparing](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onToolbarPreparing) function. Use its parameter's **toolbarOptions** field to access the [Toolbar's configuration](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/) object. This object includes the command collection: the **toolbarOptions**.[items](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/) array.
+When you enable a DataGrid feature, the component adds the corresponding control to the toolbar. You can use the [toolbar](Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/toolbar/).[items[]](Documentation/21_2/ApiReference/UI_Components/dxDataGrid/Configuration/toolbar/items/) array to customize these predefined controls and add DevExtreme components and custom elements to the toolbar. Declare the array elements in the order you want to display them in the UI:
```

Comment:
> When you enable a DataGrid feature, the component adds the corresponding control to the toolbar.

This is a description for predefined controls, we have a dedicated section for it.

```suggestion
The DataGrid includes an integrated toolbar that displays predefined and custom controls. To add or remove toolbar items, declare the [toolbar](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/toolbar/).[items[]](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/toolbar/items/) array.

This demo illustrates how to add the following items to the toolbar:
```